cirq-ft/cirq_ft/algos/state_preparation_test.py

```diff
@@ -30,22 +30,25 @@ def construct_gate_helper_and_qubit_order(data, eps):
     context = cirq.DecompositionContext(cirq.ops.SimpleQubitManager())
 
     def map_func(op: cirq.Operation, _):
-        gateset = cirq.Gateset(cirq_ft.And)
+        gateset = cirq.Gateset(cirq_ft.And, cirq_ft.LessThanEqualGate, cirq_ft.LessThanGate)
         return cirq.Circuit(
             cirq.decompose(op, on_stuck_raise=None, keep=gateset.validate, context=context)
         )
 
-    # TODO: Do not decompose `cq.And` because the `cq.map_clean_and_borrowable_qubits` currently
-    # gets confused and is not able to re-map qubits optimally; which results in a higher number
-    # of ancillas and thus the tests fails due to OOO.
+    # TODO: Do not decompose {cq.And, cq.LessThanEqualGate, cq.LessThanGate} because the
+    # `cq.map_clean_and_borrowable_qubits` currently gets confused and is not able to re-map qubits
+    # optimally; which results in a higher number of ancillas thus the tests fails due to OOO.
```

Comment:
is there an issue open for this? we definitely should have one.

Reply:
bump

Reply:
created #6197